src/sentry/models/integrations/sentry_app.py

```diff
@@ -215,3 +215,18 @@ def delete(self):
 
         SentryAppAvatar.objects.filter(sentry_app=self).delete()
         return super().delete()
+
+    def _get_redis_key(self, org_id):
+        from sentry.models import SentryAppInstallation
+
+        installations = SentryAppInstallation.objects.filter(
+            organization_id=org_id,
+            sentry_app=self,
+        )
+        if installations.exists():
+            installation = installations.first()
+            return f"sentry-app-error:{installation.uuid}"
+        return ""
+
+    def _disable(self):
+        self.update(events=[])
```

Comment:
its generally better to do: 
```
self.events = []
self.save(update_fields=["events"])
``` 
instead of `update` so any signals can `post_save` signals get fired.